tests/Application/config/packages/doctrine.yaml

```diff
@@ -12,3 +12,14 @@ doctrine:
         charset: UTF8
 
         url: '%env(resolve:DATABASE_URL)%'
+
+    orm:
+        auto_generate_proxy_classes: '%kernel.debug%'
+        auto_mapping: true
+        mappings:
+            App:
+                is_bundle: false
+                type: xml
+                dir: '%kernel.project_dir%/src/Resources/config/doctrine'
+                prefix: 'Tests\BitBag\SyliusShippingSubscriptionPlugin\Entity'
+                alias: App
```

Comment:
This configuration is required? 🤔

Reply:
This plugin requires "Product" entity extension. This configuration is necessary to run the plugin in a test environment.
Look at the `tests/Application/src/Entity/Product.php`.

Reply:
Agree 👍